keel-sql/src/main/resources/db/changelog/20200228-multi-scope-event-table.yml

```diff
@@ -0,0 +1,32 @@
+databaseChangeLog:
+  - changeSet:
+      id: multi-scope-event-table
+      author: lpollo
+      changes:
+        - renameTable:
+            newTableName: event
+            oldTableName: resource_event
+        - modifyDataType:
+            tableName: event
+            columnName: uid
+            newDataType: varchar(255)
+        - addColumn:
+            tableName: event
+            columns:
+              - column:
+                  name: scope
```

Comment:
@asher Should we add an index on `scope, uid`?

Reply:
Yes. Probably one on timestamp too since we seem to always order by that